rule_test.go

```diff
@@ -1038,6 +1038,30 @@ func TestRE(t *testing.T) {
 	}
 }
 
+func TestLastContent(t *testing.T) {
+	for _, tt := range []struct {
+		rule string
+		want string
+	}{
+		{
+			rule: `alert udp $HOME_NET any -> $EXTERNAL_NET any (sid:1337; msg:"foo"; content:"foo"; content:"bar"; within:40;)`,
+			want: `bar`,
+		},
+		{
+			rule: `alert udp $HOME_NET any -> $EXTERNAL_NET any (sid:1337; msg:"foo"; content:"bar"; content:"foo"; within:40;)`,
```

Comment:
can you add an additional test where the OrderedMatchers contain non-content types as well (PCRE, ByteMatch, etc.) to ensure that we're really returning the last Content and not something else.

Reply:
Done and added a test case for the nil return value as well